scripts/uberenv/uberenv.py

```diff
@@ -297,9 +297,10 @@ def __init__(self, opts, extra_opts):
         UberEnv.__init__(self,opts,extra_opts)
 
         # setup architecture triplet
-        self.triplet = opts["triplet"]
-        if self.triplet is None:
-           self.triplet = os.getenv("VCPKG_DEFAULT_TRIPLET", "x86-windows")
+        self.vcpkg_triplet = self.set_from_args_or_json("vcpkg_triplet", False)
+        print("triplet: {}".format(self.vcpkg_triplet))
+        if self.vcpkg_triplet is None:
+           self.vcpkg_triplet = os.getenv("VCPKG_DEFAULT_TRIPLET", "x86-windows")
```

Comment:
@agcapps -- I modified this to allow uberenv's ``project.json`` to more easily supply a ``vcpkg-triplet``

Reply:
Thanks!